### PR TITLE
UI: Show commands in command history panel

### PR DESCRIPTION
**Pull request by chloelee767**
This PR contributes towards the migration of the existing UI to our mockup. Only the command history panel on the left was modified, the changes made are:

- Executed commands (and not just their results) are now shown
- Successful and unsuccessful commands are shown in different colors

(ignore the white bar on top in the image below, it's an issue with my OS)
image

### CODE REVIEW

File: src/main/java/seedu/resireg/ui/ResultDisplay.java [hunk 1]
```diff
@@ -28,14 +27,15 @@ public ResultDisplay() {
         resultDisplayScroll.vvalueProperty().bind(responseContainer.heightProperty());
     }
 
-    public void setFeedbackToUser(String feedbackToUser) {
+    public void setFeedbackToUser(String command, String feedbackToUser, boolean isError) {
         requireNonNull(feedbackToUser);
-        TextArea response = new TextArea(feedbackToUser);
-        response.setEditable(false);
-        response.getStyleClass().add("result-display");
+        ResultDisplayItem response = new ResultDisplayItem(command, feedbackToUser, isError);
```
Comment (JingYenLoh) [inline src/main/java/seedu/resireg/ui/ResultDisplay.java:32]: Thanks for extracting this out :smile:

**Review by JingYenLoh — APPROVED** [review]
LGTM, but can you rebase on master before I merge?

**Comment by chloelee767** [comment]
> LGTM, but can you rebase on master before I merge?

Alright done!

**Comment by JingYenLoh** [comment]
Thanks!

**Pull request merged**